pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1414,6 +1418,10 @@ func TestSendfileRegression(t *testing.T) {
 
 	for _, family := range []network.ConnectionFamily{network.AFINET, network.AFINET6} {
 		t.Run(family.String(), func(t *testing.T) {
+			if family == network.AFINET6 && !isTestIPv6Enabled(cfg) {
```

Comment:
We're skipping even tcp?

Reply:
I've got a separate branch that adds config options for TCPv6 and UDPv6, so we can narrow the impact to just UDPv6.  I'm waiting on CI restoration to open the PR.

Reply:
#16843